src/diffpy/utils/transforms.py

```diff
@@ -16,6 +16,9 @@
     "The supplied q-array and wavelength will result in an impossible two-theta. "
     "Please check these values and re-instantiate the DiffractionObject with correct values."
 )
+invalid_input_emsg = (
```

Comment:
I think zero's are valid.  I could imagine many situations where I would like to give a Q all the way to zero, so I am not sure I like this.